### DISCUSSION TITLE
Will Jax implementation on Apple silicon open source in the future?

### DISCUSSION THREAD

**Discussion by gouchangjiang** [jax-ml/jax#18285] [General]
Hi, will the Jax implementation on Apple silicon be open source one day? The same question has been asked by others on Apple dev [link](https://developer.apple.com/forums/thread/739079), but no one responded. I have an m2pro MBP, I think it a good place to learn Jax and xla, and then contribute to the repository. 

I think the key is how to convert the stable HLO format JAX graphs into MPSGraph.

Thanks in advance.

**Comment by jakevdp** [comment]
cc/ @kulinseth @shuhand0
